index.html

```diff
@@ -76,6 +76,12 @@ <h1 id="title"><i class="todo">PROPOSED</i> Verifiable Credentials Working Group
 
       <p class="mission">The <strong>mission</strong> of the <a href="">Verifiable Credentials Working Group</a> is to make expressing, exchanging, and verifying credentials easier and more secure on the web.</p>
 
+      <p class="motivation">
+Readers that are new to this work should examine the <a href="https://www.w3.org/TR/vc-data-model/">latest Verifiable
```

Comment:
```suggestion
Readers that are new to this work should review the <a href="https://www.w3.org/TR/vc-data-model/">latest Verifiable
```

Reply:
*lol* -- I wish Github had a "you have 125 pending code reviews that you thought you had submitted, but you didn't." reminder on the UI.